Sources/Rendering/OpenGL/ImageMapper/index.js

```diff
@@ -929,19 +950,15 @@ function vtkOpenGLImageMapper(publicAPI, model) {
       model._openGLRenderWindow.getGraphicsResourceForObject(colorTransferFunc);
 
     const reBuildC =
-      !cTex?.vtkObj?.getHandle() ||
-      cTex?.hash !== cfunToString ||
-      model.colorTextureString !== cfunToString;
+      !cTex?.oglObject?.getHandle() || cTex?.hash !== cfunToString;
     if (reBuildC) {
+      model.colorTexture = vtkOpenGLTexture.newInstance({
```

Comment:
fyi, a `newInstance()` call is not cheap in VTK.js. 
That's why we were trying to "reuse" them.
I'm not sure why you got rid of that.

Reply:
I removed it because the render window is now the owner of the resource
Only the owner can free the texture